lib/msal-browser/src/interaction_handler/RedirectHandler.ts

```diff
@@ -41,6 +32,20 @@ export class RedirectHandler extends InteractionHandler {
         return window;
     }
 
+    navigateToRequestUrl(): void {
+        if (window.parent === window) {
+            const loginRequestUrl = this.browserStorage.getItem(TemporaryCacheKeys.ORIGIN_URI);
+
+            // Redirect to home page if login request url is null (real null or the string null)
+            if (!loginRequestUrl || loginRequestUrl === "null") {
+                this.authModule.logger.error("Unable to get valid login request url from cache, redirecting to home page");
```

Comment:
This could probably be a warning (since nothing is actually breaking).

Reply:
(And yes I am aware I originally wrote this logic :) )

Reply:
done.